contrib/helm/harbor/templates/registry/registry-ss.yaml

```diff
@@ -44,11 +44,9 @@ spec:
           mountPath: /etc/registry/config.yml
           subPath: config.yml
       volumes:
-{{- if not .Values.registry.objectStorage }}
-{{- if not .Values.persistence.enabled }}
+{{- if or (not .Values.persistence.enabled) .Values.registry.objectStorage }}
```

Comment:
this could be simply `{{- if not .Values.registry.objectStorage }}`. Since `.Values.persistence.enabled` is not used other places related to registry storage. Sorry for the original confusing code.

Reply:
Also need to add `{{- if not .Values.registry.objectStorage }}` around line 38-39

Reply:
Thanks very much for this PR.

Reply:
BTW: suggest change the commit tile to "[Chart] Fix registry storage config issue"

Reply:
i've renamed the PR

=)